feature-flags/feature-flags.development.ts

```diff
@@ -3,14 +3,10 @@ import type { FeatureFlagsInterface } from './feature-flags.interface';
 export const flagsDevelopment: FeatureFlagsInterface = {
   FEATURE_CARD_NAVIGATION: true,
   FEATURE_SITEMAP: process.env.ENABLE_SITEMAP === 'true',
-  FEATURE_TRANSPARENCY_NEW_TABLE: true,
-  FEATURE_CU_INDEX_NEW_TABLE: true,
-  FEATURE_CU_ABOUT_NEW_CONTAINER: true,
-  FEATURE_TRANSPARENCY_COMMENTS: true,
   FEATURE_GLOBAL_ACTIVITIES: true,
   FEATURE_AUTH: true,
   FEATURE_MKR_VESTING: true,
-  FEATURE_AUDIT_REPORTS: false,
+  FEATURE_AUDIT_REPORTS: true,
```

Comment:
this feature flag should the `false` value

Reply:
you are right